src/views/tutorial-library/components/pagination/index.tsx

```diff
@@ -0,0 +1,136 @@
+import { IconArrowLeft16 } from '@hashicorp/flight-icons/svg-react/arrow-left-16'
```

Comment:
🙋🏻 Is there any documentation from algolia explaining why their examples have `<a>` elements being used with `preventDefault`, rather than using a `<button>` element? It feels really strange that we have to prevent default behavior for the links.

Reply:
I believe it's because the links are still valid on their own, and we are treating the result as a navigation in the sense that we update the URL and the content of the page, we're just doing it client-side instead of relying on the browser (much like next's routing works, for example).

Reply:
The URL is updated as a result of the search state change, in this case through `refine()`.

Reply:
Got it! After doing some more reading yesterday for Global Search, I saw how the `createURL` function works too. 😊